lib/components/app/desktop-nav.tsx

```diff
@@ -91,12 +105,14 @@ const DesktopNav = ({
 
           <ViewSwitcher sticky />
 
-          <Nav pullRight>
+          <StyledNav pullRight>
             {popupTarget && (
               <NavItemOnLargeScreens
                 onClick={() => setPopupContent(popupTarget)}
+                title={popupButtonText}
               >
-                <FormattedMessage id={`config.popups.${popupTarget}`} />
+                <Icon iconType={popupTarget} />
```

Comment:
You have the chat bubble icon in the screenshot and I think I prefer it to this popout icon. We use this popout icon in other places to indicate that something will open in a new window/tab which feels a little inconsistent with the survey iframe behavior.

Reply:
I am not sure to understand... It's the same (configured) icon that is also used at the top of itinerary results, at the bottom of an expanded itinerary, and in the app menu. Maybe the a11y text needs to say "Opens in a popup" or something?

Reply:
I don't think we need additional text! Using this icon for the feedback button in the app menu makes sense, because in mobile view this will open a brand new window in the browser. But in desktop view, it only opens a modal. And since generally when we use this icon (https://github.com/opentripplanner/otp-react-redux/pull/859) it means it's opening a new window, it seems more consistent to have a different icon for the desktop nav.

I suppose I don't mind keeping this icon for both, I can defer to the second reviewer! But I do prefer the chat bubble, if that's an option.

Reply:
Per our call earlier, I will add additional a11y text to warn of a tab/window opening on mobile. We will keep using any configured icon if any.

Reply:
I added the a11y text in https://github.com/opentripplanner/otp-react-redux/pull/903/commits/63ba3ffd4a11cee222f58914c03cabf8a1bc1bd9 and subsequent commits.